packages/components/src/menu/stories/index.story.tsx

```diff
@@ -20,10 +20,10 @@ import Button from '../../button';
 import Modal from '../../modal';
 import { createSlotFill, Provider as SlotFillProvider } from '../../slot-fill';
 import { ContextSystemProvider } from '../../context';
-import type { MenuProps } from '../types';
+import type { Props } from '../types';
 
 const meta: Meta< typeof Menu > = {
-	id: 'components-experimental-menu',
+	id: 'components-menu',
```

Comment:
This change was necessary because the generated README links to `?path=/docs/components-menu--docs`. I've updated `storybook/manager-head.html` to add a redirect from the old experimental URL.